CatLover.Api/controllers/breed.controller.go

```diff
@@ -0,0 +1,45 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"catlover.api/models"
+	"catlover.api/services"
+
+	"github.com/gin-gonic/gin"
+)
+
+type ResponseBreedById struct {
+	Breed models.Breed
+	Cats  []models.Cat
+}
+
+// GET /breeds (Get all Breeds)
+func GetAllBreeds(c *gin.Context) {
+	var breeds = services.GetAllBreeds()
+	fmt.Println("Endpoint Hit: GetAllBreeds")
```

Comment:
I think this line should be the first one in the controller, WDYT?

Reply:
Correct, this line must be the first in every controller, in order to know that if the controller has been hit, in case the second line (var breed...) or any other line gets an error. By mistake i forgot the Print in the second line. Same mistake and in the CatController.